src/pages/CourseGraph/style.scss

```diff
@@ -0,0 +1,37 @@
+@import url('normalize.css/normalize.css');
+
+.CourseGraph {
```

Comment:
This line is breaking the CI, either disable the linting for this line, change the linting rules if you think that's better, or adjust the class name. I think to begin with an id makes more sense anyways